src/MapViewController.swift

```diff
@@ -446,6 +451,9 @@ open class MapViewController: UIViewController, LocationManagerDelegate {
     findMeButton.setBackgroundImage(UIImage(named: "ic_find_me_pressed"), for: [.selected])
     findMeButton.backgroundColor = UIColor.white
     findMeButton.autoresizingMask = [.flexibleTopMargin, .flexibleLeftMargin]
+    let viewRect = mapView.bounds
```

Comment:
This won't work correctly. The `createFindMeButton()` function is called in the initializer of the VC. Referencing the view at this stage in the lifecycle will force view generation to happen outside of the proper lifecycle events. 

What are you attempting to solve by moving this around?